integreat_compass/core/middleware/access_control_middleware.py

```diff
@@ -40,6 +40,7 @@ def __call__(self, request):
         if (
             namespaces.PUBLIC not in resolver_match.namespaces
             and not request.user.is_authenticated
+            and not "admin" in request.path
```

Comment:
```suggestion
            and "admin" not in resolver_match.namespaces
```
We don't want this to happen for e.g. `/offers/badminton-course/` :joy:

Reply:
Hahaha, good point 😂